corehq/apps/fixtures/upload/run_upload.py

```diff
@@ -248,3 +228,35 @@ def _process_item_field(field, data_item):
     return FieldList(
         field_list=field_list
     )
+
+
+def _process_data_item(domain, replace, data_type, di, item_fields, item_attributes, sort_key):
```

Comment:
`di` => `data_item`? (assuming that's what it is...I can't tell and didn't dig around in the calling code)

Reply:
Kind of. Its a `FixtureDataItem`'s row in the table (though maybe it has some other attributes as well). I think I kept it as di as that was what the old code was using throughout (and what was in the for loop). I also thought about changing the `old_data_item` and `new_data_item` in this function, but wanted to keep this PR single purpose of moving around code as much as possible